objects/ohitbox/Step_2.gml

```diff
@@ -22,228 +22,400 @@ if (!global.game_paused)
 			{
-				// Set the correct states for the attacker and victim
-				owner.state = eState.HOLD;
-				owner.animTimer = 0;
-				
-				collision_list[| i].owner.state = eState.BEING_GRABBED;
-				collision_list[| i].owner.sprite_index = collision_list[| i].owner.CharacterSprites.hurt_Sprite;
-				collision_list[| i].owner.animTimer = 0;
-				collision_list[| i].owner.x = owner.x + (attackProperty.holdXOffset[hitboxID] * owner.image_xscale);
-				collision_list[| i].owner.isShortHopping = false; // Make sure the victim is not using their shorthop fall speed.
-				owner.heldOpponent = collision_list[| i].owner;
-				
-				// Multiple hitboxes
-				hasHit = true;
-				ds_list_add(collision_list[| i].owner.hitByGroup, attackProperty.group[hitboxID]);
-				
-				// Depth Sorting
-				owner.depth = -1;
-				collision_list[| i].owner.depth = 0;
-				
-				// Reset Frame Advantage Counter
-				oGameManager.frameAdvantage = 0;
-				
-				var particle = instance_create_layer(collision_list[| i].owner.x, collision_list[| i].owner.y, "Particles", oParticles);
-				with (particle) 
+				//Set who the player is currently targeting
+				owner.target = collision_list[| i].owner.id;
+			
+				// Throw Teching
+				if (attackProperty.attackType[hitboxID] == eAttackType.GRAB && (collision_list[| i].owner.state == eState.GRAB || collision_list[| i].owner.state == eState.HOLD) && collision_list[| i].owner.animTimer <= 8)
 				{
-					lifetime = 10;
-					sprite_index = sHitEffect;
-					image_xscale = other.owner.image_xscale * -1;
-				}
+					owner.state = eState.THROW_TECH;
+					owner.hsp = -2 * owner.image_xscale;
+					owner.animTimer = 0;
 				
-			} else 
-			// BLOCKING                                  Check to see if we are blocking correctly
-			if (collision_list[| i].owner.canBlock) && ((attackProperty.attackType[hitboxID] == eAttackType.LOW && collision_list[| i].owner.verticalMoveDir == -1) || attackProperty.attackType[hitboxID] == eAttackType.MID || (attackProperty.attackType[hitboxID] == eAttackType.HIGH && collision_list[| i].owner.verticalMoveDir != -1)){
-					
-				collision_list[| i].owner.prevState = eState.BLOCKING;
-				collision_list[| i].owner.state = eState.HITSTOP; // Set the player's state to hitstop
+					collision_list[| i].owner.state = eState.THROW_TECH;
+					collision_list[| i].owner.hsp = -2 * collision_list[| i].owner.image_xscale;
+					collision_list[| i].owner.animTimer = 0;
 				
-				owner.prevState = owner.state; // Set the owner's previous state
-				owner.state = eState.HITSTOP;
+					// Meter Build - Both players get some meter
+					collision_list[| i].owner.superMeter += 5;
+					owner.superMeter += 5;
 				
-				// Handle if the opponent is Crouch blocking or not
-				if (collision_list[| i].owner.verticalMoveDir == -1)
-				{
-					collision_list[| i].owner.sprite_index = sRussel_Crouch_Block;
-					collision_list[| i].owner.isCrouchBlocking = true;
-				}
-				else 
+					hasHit = true;
+				} else

```

Comment:
I'm not sure what the intent is here, but the result is likely to be that the next if is actually an else if.

If this is intended, this else should be moved down to line 73 and placed before the if. If this is not the intention, this else should be removed.